ur5e_moveit_config/launch/run_move_group.launch.py

```diff
@@ -0,0 +1,145 @@
+    ompl_planning_pipeline_config['move_group'].update(ompl_planning_yaml)
+
+    # Trajectory Execution Functionality
+    controllers_yaml = load_yaml('run_move_group', 'config/controllers.yaml')
+    moveit_controllers = { 'moveit_simple_controller_manager' : controllers_yaml,
+                           'moveit_controller_manager': 'moveit_simple_controller_manager/MoveItSimpleControllerManager'}
+
+    trajectory_execution = {'moveit_manage_controllers': True,
+                            'trajectory_execution.allowed_execution_duration_scaling': 1.2,
+                            'trajectory_execution.allowed_goal_duration_margin': 0.5,
+                            'trajectory_execution.allowed_start_tolerance': 0.01}
+
+    planning_scene_monitor_parameters = {"publish_planning_scene": True,
+                 "publish_geometry_updates": True,
+                 "publish_state_updates": True,
+                 "publish_transforms_updates": True}
+
+    # Start the actual move_group node/action server
+    run_move_group_node = Node(package='moveit_ros_move_group',
+                               executable='move_group',
+                               output='screen',
+                               parameters=[robot_description,
+                                           robot_description_semantic,
+                                           kinematics_yaml,
+                                           ompl_planning_pipeline_config,
+                                           trajectory_execution,
+                                           moveit_controllers,
+                                           planning_scene_monitor_parameters])
+
+    # RViz
+    rviz_config_file = get_package_share_directory('ur_description') + "/cfg/view_robot.rviz"
+    rviz_node = Node(package='rviz2',
+                     executable='rviz2',
+                     name='rviz2',
+                     output='log',
+                     arguments=['-d', rviz_config_file],
+                     parameters=[robot_description,
+                                 robot_description_semantic,
+                                 ompl_planning_pipeline_config,
+                                 kinematics_yaml])
+
+    # Static TF
+    static_tf = Node(package='tf2_ros',
+                     executable='static_transform_publisher',
+                     name='static_transform_publisher',
+                     output='log',
+                     arguments=['0.0', '0.0', '0.0', '0.0', '0.0', '0.0', 'world', 'base_link'])
+
+    # Publish TF
+    robot_state_publisher = Node(package='robot_state_publisher',
+                                 executable='robot_state_publisher',
+                                 name='robot_state_publisher',
+                                 output='both',
+                                 parameters=[robot_description])
+
+# TODO(andyz): fake_joint_driver conflicts with the current ros2_controller commits
```

Comment:
May I ask what is this `fake_joint_driver`? Is this something I can help within `ros2_control`?

Reply:
No, it is inherited from the moveit2 release which was locked on commits of fake_joint_driver ros2 fork, ros2_control and ros2_controllers. This repository is using latest versions of ros2_control packages which are incompatible with the fake_joint_driver...

Reply:
do we need something like this in the future? I believe so :) I am planning to implement something in this direction anyway to enable testing of robots' URDFs and controller without dynamic and  Gazebo